tc/proto/mapping_options.proto

```diff
@@ -44,13 +44,12 @@ message MappingOptionsProto {
   optional TilingProto tiling = 4;
   // Unroll innermost loops to until at most "unroll" copies.  If not provided,
   // do not unroll.
-  optional uint64 unroll = 7;
-  // Apply tiling to imperfectly nested loops if possible.  Same tilesizes are
-  // used.
-  required bool tile_imperfectly_nested = 8;
-  // reserved 5, 6, 9 to 13; can only activate with proto3
-  // Reserved: 5, 6, 9-13 -> factored out into CudaMappingOptionsProto
-  required bool match_library_calls = 14;
+  optional uint64 unroll = 5;
```

Comment:
I think we did not want to reuse the numbers between different versions of proto, otherwise it attempts to match incompatible messages.

Reply:
That's in a world where proto is stable enough, I'd claim we're still in alpha stage and we don't save any protos.

Reply:
I would go for a minimally intrusive change in this case, meaning don't change the numbers unless you have to